### DISCUSSION TITLE
How to Implement Openthread Border Router in RTOS like Zephyr, FreeRTOS..?

### DISCUSSION THREAD

**Discussion by ravimannem123** [openthread/openthread#10523] [Q&A]
Is there any specific implementation plan for some simple applications, such as using embedded operating systems such as FreeRTOS or Zephyr with LWIP to implement a Open Thread Border Router (OTBR). Any ideas..?

**Comment by jwhui** [comment]
Espressif has an RTOS-based solution: https://openthread.io/guides/border-router/espressif-esp32

**Reply by jchellap** [reply]
Hi Jonathan
Thanks for the link!
How about Zephyr RTOS with open thread? Any references?

**Reply by jwhui** [reply]
I would reach out to the Zephyr community directly.

**Reply by ravimannem123** [reply]
Hi jwhui,

Could you please share the info about Open Thread with Zephyr RTOS implementation references ?

**Reply by jwhui** [reply]
As I mentioned above, I would reach out to the Zephyr community directly.

**Reply by ravimannem123** [reply]
Hi @jwhui ,

Any update about the OPenThread with Zephyr RTOS implementation references, if so could you please share the info ?

**Reply by jwhui** [reply]
As I mentioned above, I would reach out to the Zephyr community directly.
